src/test/java/io/openliberty/tools/intellij/it/UIBotTestUtils.java

```diff
@@ -1596,6 +1596,7 @@ public static void runActionFromSearchEverywherePanel(RemoteRobot remoteRobot, S
                 JTextFieldFixture searchField = projectFrame.textField(JTextFieldFixture.Companion.byType(), Duration.ofSeconds(10));
                 searchField.click();
                 searchField.setText(action);
+                TestUtils.sleepAndIgnoreException(1); // allow the search for 'action' to complete
```

Comment:
This is the exact same line changed in https://github.com/OpenLiberty/liberty-tools-intellij/pull/771. This will create a merge conflict since you have a different comment here.

Reply:
So it is. The change in SingleModMPProjectTestCommon.java is not needed so I will cancel this pull request.